api/src/opentrons/protocols/api_support/constants.py

```diff
@@ -4,5 +4,6 @@
 
 OPENTRONS_NAMESPACE = "opentrons"
 CUSTOM_NAMESPACE = "custom_beta"
-STANDARD_DEFS_PATH = Path("labware/definitions/2")
+# STANDARD_DEFS_PATH = Path("labware/definitions/2")
+STANDARD_DEFS_PATH = Path("labware/definitions")
```

Comment:
Makes sense, but let's just delete the old code. No reason to keep it around.

```suggestion
STANDARD_DEFS_PATH = Path("labware/definitions")
```